util/buildRelease/gen_release

```diff
@@ -166,6 +166,7 @@ SystemOrDie("make man-chpldoc");
 SystemOrDie("make clobber");
 
 print "Creating the examples directory...\n";
+SystemOrDie("rm -rf examples"); # remove examples symbolic link
```

Comment:
You don't want "rm examples" (without "-f") because if examples does not exist (and usually it won't) then "rm" will return an error (which in this case probably won't be ignored, because "SystemOrDie"). 
IMO, "rm -f examples" comes closest to Brad's intent.